scripts/apps/workspace/content/views/profile-settings.html

```diff
@@ -146,7 +146,7 @@ <h3 class="modal__heading">{{ :: 'Editing' | translate }} "{{ editing.form.label
         </form>
     </div>
     <div class="modal__footer">
-        <button id="profile-update" class="btn btn--primary" ng-disabled="editForm.$invalid || !editForm.$dirty" ng-click="ctrl.update()" translate>Save</button>
+        <button id="profile-update" class="btn btn--primary" ng-disabled="editForm.$invalid || !editForm.$dirty || !ctrl.savingInProgress" ng-click="ctrl.update()" translate>Save</button>
```

Comment:
> `|| !ctrl.savingInProgress`

Should exclamation mark be here? It should be disabled when saving **IS** in progress.

Reply:
my bad, yes you are right I have added it mistakenly.